mypy.ini

```diff
@@ -0,0 +1,4 @@
+[mypy]
+
+[mypy-mcp.*]
+ignore_missing_imports = True
```

Comment:
Can we remove this file and move this to our pyproject.toml

Reply:
https://github.com/livekit/agents/blob/8ff1efa7d7543c1e9f07f9b1b27da3a5e870968b/pyproject.toml#L77